### PR TITLE
DCM 62: Improve manage mappings UI to handle all mapping related functions

### PR DESCRIPTION
**Pull request by Gravewalker666**
### Purpose

fix the ticket [[DCM-62] Improve manage mappings UI to handle all mapping related functions](https://issues.openmrs.org/browse/DCM-62)

### Approach

- Remove uploadMappings and exportMappings user interfaces
- Add import mappings and export mappings functions to the manage mappings UI

### Demo

https://user-images.githubusercontent.com/45477334/128125253-7498e9c4-414c-4dd0-b16c-efd603f8ede8.mov

### Environment

- Java 8 Oracle
- Maven 3.6.3
- OpenMRS Reference Application 2.11.0
- MySQL server
- macOS bigSur

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/dhisconnector/web/controller/DHISConnectorController.java [hunk 1]
```diff
@@ -332,6 +304,28 @@ public void manageMappings(ModelMap model) {
 		model.addAttribute("showLogin", (Context.getAuthenticatedUser() == null) ? true : false);
 	}
 
+	@RequestMapping(value = "/module/dhisconnector/manageMappings", method = RequestMethod.POST)
+	public void manageMappings(ModelMap model,
+							  @RequestParam(value = "mapping", required = false) MultipartFile mapping) throws IOException {
```
Comment (HerbertYiga) [inline omod/src/main/java/org/openmrs/module/dhisconnector/web/controller/DHISConnectorController.java:309]: did this line formatting come in by mistake?

File: omod/src/main/webapp/manageMappings.jsp [hunk 2]
```diff
@@ -41,8 +58,18 @@
...
 		</table>
+		<div class="margin-top">
+			<form class="inline-block" method="POST" action="exportMappings.form">
+				<input type="hidden" value="" name="selectedMappings" id="selected-mappings-to-export">
+				<input ng-disabled="disableMultipleActions()" type="submit" value='Export Selected'>
+			</form>
+			<input class="inline-block" type="button" value="<openmrs:message code='dhisconnector.manageMappings.deleteSelected'/>"
+				   ng-disabled="disableMultipleActions()"
+				   ng-click="deleteSelectedMappings()"
+			>
```
Comment (HerbertYiga) [inline omod/src/main/webapp/manageMappings.jsp:69]: do we need this?

Comment (Gravewalker666) [reply omod/src/main/webapp/manageMappings.jsp:69]: I'll move it up

File: omod/src/main/java/org/openmrs/module/dhisconnector/web/controller/DHISConnectorController.java (hunk 1 above)
Comment (Gravewalker666) [reply omod/src/main/java/org/openmrs/module/dhisconnector/web/controller/DHISConnectorController.java]: I'll reformat it, thank you

Comment (Gravewalker666) [reply omod/src/main/java/org/openmrs/module/dhisconnector/web/controller/DHISConnectorController.java]: Resolved with commit 457f8cf

File: omod/src/main/webapp/manageMappings.jsp (hunk 2 above)
Comment (Gravewalker666) [reply omod/src/main/webapp/manageMappings.jsp]: Resolved with commit 457f8cf

**Pull request merged**